app/views/core_induction_programmes/mentor_material_parts/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= govuk_breadcrumbs breadcrumbs: mentor_material_breadcrumbs(current_user, @mentor_material_part.mentor_material) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">Content change preview</h1>
+    <%# <%= govuk_link_to "Delete lesson part", lesson_part_show_delete_path(lesson_part_id: @course_lesson_part.id), button: true, class: "govuk-button--secondary"  %1> %>
```

Comment:
Hmmm. I'm happy to add it in, although probably needs to be switched to delete mentor material parts.

Reply:
![image](https://user-images.githubusercontent.com/472830/122893242-0f1e2900-d33e-11eb-8eb6-7578d26ca87f.png)

😠😠😠😠😠

Reply:
Hehe, sorry, I just go through everything new in the morning and your PR got reviewed as part of that 😂